File: ARCADE/The Core/Level 1_7: lateRide.cs
    using System;
     
    public class Test
    {
    	public static int lateRide(int n){
            int result = 0;
            result = result + sum(n / 60);
            result = result + sum(n % 60);
            return result;
        }
     
        public static int sum(int a){
            return a / 10 + a % 10;
        }
    	public static void Main()
    	{
    		int n = 240;
    		Console.WriteLine(lateRide(n));
    		int m = 808;
    		Console.Write(lateRide(m));
    	}
    }

File: ARCADE/The Core/Level 1_7: lateRide.pas
program lateRide;

function sum(n: integer): integer;
begin
    sum := n div 10 + n mod 10;
end;

function lateRide(n: integer): integer;
var rs : integer;
begin
    rs := 0;
    rs := rs + sum(n div 60);
    rs := rs + sum(n mod 60);
    lateRide := rs;
end;

var n : integer;
begin
	n := 240;
	writeln(lateRide(n));
	n := 808;
	writeln(lateRide(n));
end.
